docs/modules/ROOT/pages/guides/hrmp_channels.adoc

```diff
@@ -0,0 +1,147 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Sending Cross-Chain Messages between Parachains
+
+The supported way to exchange cross-chain messages (XCM) between parachains is to use Horizontal Relay-routed Message Passing (HRMP) channels.
+
+Each HRMP channel is unidirectional. In order to enable full connectivity between two parachains, two HRMP channels must be opened: one for sending outgoing XCM and the other for receiving incoming XCM.
+
+== Opening an HRMP Channel
+
+Opening a channel between two parachains A and B takes 2 steps:
+1. parachain A initiates a channel request
+2. parachain B accepts the channel request
+
+For step (1), parachain A calls `hrmp > hrmpInitOpenChannel(recipient, proposedMaxCapacity, proposedMaxMessageSize)` to create a channel request with the input configuration.
+
+For step (2), parachain B calls `hrmp > hrmpAcceptOpenChannel(sender)` to accept the channel open request from the input sender.
+
+In order to dispatch a call from its sovereign origin, a parachain may use governance to send the encoded call in a Transact instruction to the Relay Chain, but it may also execute this logic autonomously (e.g. on the notification that a channel was requested).
+
+== Connecting to Ecosystem Parachains
+
+The examples in this section include steps to connect to existing parachains in the Polkadot ecosystem. Examples include the following Polkadot parachains:
+1. AssetHub supports managing asset balances as well as the execution of cross-chain transfers.
+2. Snowbridge supports sending ERC20 tokens between Polkadot parachains and Ethereum in both directions.
+3.  HydraDX supports DeFi functionality to provide liquidity to Polkadot.
+
+For all of the examples:
+. `paraId` for the user's parachain is set to `43211234` for all examples below.
+. `proposedMaxCapacity` and `proposedMaxMessageSize` are set to the values of Polkadot `config.hrmpChannelMaxCapacity = 1000` and `config.hrmpChannelMaxMessageSize = 102400`, respectively.
+
+=== AssetHub
+
+AssetHub supports managing asset balances as well as the execution of cross-chain transfers.
+
+AssetHub is a common-good, system parachain and is therefore controlled by relay chain governance. This means it is sufficient to propose opening both channels at once through the relay chain's `GeneralAdmin` track. The proposal should set its call (proposed to be executed) to `utility.batchAll` with the input including 2 calls to `hrmp.forceOpenHrmpChannel` to open a channel in each direction between the user parachain and AssetHub.
+
+The first call to `hrmp.forceOpenHrmpChannel` proposes opening a unidirectional channel to send XCM from the user parachain to AssetHub. If AssetHub's paraId is set to `1000`, here are the inputs:
```

Comment:
since we are using `1000` for the user's parachain, it would be less confusing to use other values for the target parachain. I thought I misread and re-read the explanations carefully again

Reply:
or, since AssetHub's paraId is actually `1000` in reality, we can use another value for the user's parahcain. That would be the least confusing option imo